hw3_2/hw3-2.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,85 @@
+namespace hw3_2.Tests;
+

```

Comment:
```suggestion

using NUnit.Framework;

```
А у вас без `using` тесты запускались?